pkgs/games/factorio/default.nix

```diff
@@ -175,10 +175,11 @@ let
     headless = base;
     demo = base // {
 
-      buildInputs = [ makeWrapper ];
+      buildInputs = [ makeWrapper libpulseaudio ];
 
       libPath = stdenv.lib.makeLibraryPath [
         alsaLib
+        libpulseaudio
```

Comment:
This addition probably means 17.x won't also need alsaLib. Can you check that, and if it's true, add a TODO to remove it when stable is updated?